lib/LTP/utils.pm

```diff
@@ -150,7 +150,7 @@ sub export_ltp_env {
 
 # Set up basic shell environment for running LTP tests
 sub prepare_ltp_env {
-    assert_script_run('export LTPROOT=' . get_ltproot() . '; export LTP_COLORIZE_OUTPUT=n PATH=$LTPROOT/testcases/bin:$PATH');
+    assert_script_run('export LTPROOT=' . get_ltproot() . '; export LTP_COLORIZE_OUTPUT=n TMPDIR=/tmp PATH=$LTPROOT/testcases/bin:$PATH');
```

Comment:
BTW safer would have been `TMPDIR=${TMPDIR:-/tmp}`.

Reply:
For a record, @mdoucha find that https://github.com/linux-test-project/ltp/blob/master/runtest/fs requires that.